copyright

```diff
@@ -1,4 +1,4 @@
-﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
There seem to be quite a few unrelated changes in here, like this one.

Reply:
Pretty sure this is another UTF-8 BOM getting chomped (or readded). A quick look at the format spec doesn't indicate that the BOM is required or disallowed.

Reply:
Actually I have no idea what happened in this line.

Reply:
If you are using the desktop app, then you can deselect the top thing and it won't be in the update

Reply:
No, I don't use desktop apps, I make edits directly in the browser. Anyone knows how to revert this line? (Or via a terminal command?)

Reply:
Use something better than the browser. Then you can name your branches too